SB2018-volunteer/2018-04-13.md

```diff
@@ -3,23 +3,25 @@ to: All Volunteers
 subject: Volunteer Information - SourceBots 2018
 ---
 
-First of all, on behalf of everyone at SourceBots we would like to say a **massive** thank you for volunteering at the competition! This event could not run without you and we appreciate the hardwork and effort that you will put in!
+First of all, on behalf of everyone at SourceBots we would like to say a **massive** thank you for volunteering at the competition, even though it will be hard work, we can assure you that it will be a fun weekend filled with robot madness and endless fun.
 
-Even though it is hard work, we can assure you that it will be a fun weekend filled with robot madness and making sure everything is running smoothly!
-
-To that end, we thought now would be a good time to send an email detailing information about the competition! :)
+To that end, we thought now would be a good time to send an email detailing information about the competition!
 
 ### Date and location
 
-As a reminder, the competition dates are 21st April - 22nd April and it will be held in [The Cube (SUSU Building)](http://data.southampton.ac.uk/building/42.html).
+As a reminder, the competition dates are 21st April - 22nd April and it will be held in [The Cube (SUSU Building)](http://data.southampton.ac.uk/building/42.html) which is on the bottom floor of the Students' Union building.
 
-### Roles
+This will take up the whole day so if you're around please drop in between **0800-2300**.
```

Comment:
I doubt it's useful if they drop in at 2259

Reply:
potentially a word change, saying we'll be working between these times